corehq/apps/reports/standard/project_health.py

```diff
@@ -241,15 +241,36 @@ def get_newly_performing(self):
             return sorted(dropouts, key=lambda stub: -stub.delta_forms)
 
 
-class ProjectHealthDashboard(ProjectReport):
+class ProjectHealthDashboard(ProjectReport, DatespanMixin):
     slug = 'project_health'
-    name = ugettext_noop("Project Performance")
+    name = ugettext_lazy("Project Performance")
     base_template = "reports/project_health/project_health_dashboard.html"
 
     fields = [
         'corehq.apps.reports.filters.location.LocationGroupFilter',
+        'corehq.apps.reports.filters.dates.DatespanFilter'
```

Comment:
I'm using `DatespanFilter` to set `startdate` and `enddate` params, since they are used in email report template `report_email_content.html`. Would love some feedback on this workaround